pipelines/build/common/config_regeneration.groovy

```diff
@@ -513,6 +517,7 @@ return {
     String javaVersion,
     Map<String, Map<String, ?>> buildConfigurations,
     Map<String, ?> targetConfigurations,
+    String activeNodeTimeout,
```

Comment:
Can be removed if we're removing if we're not specifying a node timeout in the regen jobs

Reply:
Sorted.